### DISCUSSION TITLE
I Dont know how to install the server new start. do you have a video explaining ?

### DISCUSSION THREAD

**Discussion by Wezler** [Yaro2709/New-Star#42] [Q&A]
I Dont know how to install it. do you have a video explaining ?

cheers

**Comment by Wezler** [comment]
--
Message: unlink(includes/ENABLE_INSTALL_TOOL): Permission denied 			File: /install/index.php 			Line: 52 			URL: http://localhost:8080/New-Star-master/install/index.php 			PHP-Version: 8.0.10 			PHP-API: apache2handler 			Version: 2.7.2 (FILE) 			Debug Backtrace:#0 [internal function]: errorHandler(2, 'unlink(includes...', 'FILEPATH ...', 52) #1 /install/index.php(52): unlink('includes/ENABLE...') #2 {main}

**Comment by Yaro2709** [comment]
Try the php version below. If it doesn't work, then you have problems with php modules.
